File: apps/api/src/Features/Cuenta/Models/CuentaModel.cs
namespace Features.Cuenta.Models;

public class Cuenta
{
    public int IdCuenta { get; set; }
    public string Nombre { get; set; } = string.Empty;
    public string Tipo { get; set; } = string.Empty;
    public bool Activa { get; set; }
    public DateTime Created { get; set; }
    public DateTime Updated { get; set; }
}

File: apps/web/src/app/features/cuenta/models/cuenta.model.ts
export interface Cuenta {
  idCuenta: number;
  nombre: string;
  tipo: string;
  activa: boolean;
  created: string;
  updated: string;
}

export interface CuentaCreate {
  nombre: string;
  tipo: string;
  activa: boolean;
}

export interface CuentaUpdate {
  nombre: string;
  tipo: string;
  activa: boolean;
}

export interface CuentaSaldo {
  idCuenta: number;
  nombre: string;
  saldoActual: number;
}